### DISCUSSION TITLE
Problem with the Twitch Leecher

### DISCUSSION THREAD

**Discussion by grafsk1y** [community/community#35731] [Other Feature Feedback, Questions, & Ideas] [answered]
When I go to my Twitch account in the 
![634523](https://user-images.githubusercontent.com/115526468/195010192-eacdc903-87a1-499c-a2a2-3754bb571919.jpg)
app, I get thrown out and writes that it is impossible to connect try again

**Comment by dracco1993** [comment]
Hi!  I think you've landed in the wong place, but that's okay!

Check out this post by the Twitch Leecher developer about your issue:
https://github.com/Franiac/TwitchLeecher/issues/803

Hopefully that helps!

**Comment by otter-computer** [comment, marked as answer]
Hi @grafsk1y,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this specific project. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
